### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#153] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-17 184625](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d1290860-bff2-4b2f-892c-c114e161c6c5)

**Comment by CasuFrost** [comment]
### Idea di soluzione
_(Non è una soluzione completa, è un idea sul come procedere, ma onestamente potrei aver detto qualche castroneria senza essemene reso conto)_\
\
Come prima cosa, caratterizziamo gli omomorfismi da $\mathbb{Z}_8\rightarrow\mathbb{Z}_8$, sappiamo che un omomorfismo deve mappare l'unità  nell'unità , quindi i diversi omomorfismi di $f:\mathbb{Z}_8\rightarrow\mathbb{Z}_8$ potranno mappare $[1]$ in uno dei diversi elementi che fungono da unità in  $\mathbb{Z}_8$, ossia : $\mathcal{U}(\mathbb{Z}_8)=\{[1],[3],[5],[7]\}$\
D'altronde questi sono anche isomorfismi, e sappiamo già che $Aut(\mathbb{Z}_8)$ è isomorfo a $\mathcal{U}(\mathbb{Z}_8)$.
Prendiamo una qualsiasi applicazione $f\in Aut(\mathbb{Z}_8)$, il suo nucleo, contiene gli elementi mappati nell'unità, abbiamo già detto 
che una qualsiasi applicazione mappa $[1]$ nell'unità, quindi, $\forall f\in Aut(\mathbb{Z}_8)$ si ha che $Kerf=${[1]}. Quindi suppongo (se non ho fatto errori stupidi di logica) che tutti i gruppi quoziente $\mathbb{Z}_8/Kerf$ sia identici, a prescindere da $f$. \
Precisamente, se il nucleo è composto dal solo $[1]$, le classi laterali saranno tutte : \
\
$[1]Kerf=${[1]+[1]=[2]}\
$[2]Kerf=${[2]+[1]=[3]}\
$[3]Kerf=${[3]+[1]=[4]}
$\vdots$
$[8]Kerf=${[8]+[1]=[9]= [1]}
\
Questo fa notare che la cardinalità di $\mathbb{Z}_8/Kerf$ sia identica alla cardinalità di $\mathbb{Z}_8$, adesso possiamo applicare il teorema fondamentale di omomorfismo di gruppi, sappiamo che $\mathbb{Z}_8/Kerf$ è isomorfo a $Im(f)$, ma $f$ è una biezione ed ha 
il nucleo banale, $Im(f)=\mathbb{Z}_8$ quindi possiamo dire (suppongo?) che $\mathbb{Z}_8/Kerf$ sia isomorfo a $\mathbb{Z}_8$ qualsiasi sia $f$?
\
La dimostrazione si basa sul fatto che tutti gli omomorfismi da $\mathbb{Z}_8\rightarrow\mathbb{Z}_8$ sono isomorfismi, quindi il nucleo è sempre banale. Questo fatto non l'ho dimostrato. Ho fatto palesemente altri errori ma posto comunque in quanto può essere una vaga 
idea sul come procedere.

**Comment by cornflexxx** [comment]
Partiamo cercando di definire delle applicazioni  $\mathbb{Z}_8\to\mathbb{Z}_8$, che vadano a mappare elementi del primo gruppo in dei sottogruppi dzi $\mathbb{Z}_8$, che siano anche degli omomorfismi. Sappiamo che i sottogruppi propri di $\mathbb{Z}_8$ sono generati da tutti gli elementi che non sono coprimi con $8$, il discorso si può benissimo generalizzare ad $n$. Quindi abbiamo 3 tipi  di sottogruppi con ordine minore rispetto a quello di $\mathbb{Z}_8$:

- $\left\langle  2\right\rangle=\left\langle 6 \right\rangle=\\{[2],[4],[6],[8]\\}$ con ordine $4$ che divide $8$.
- $\left\langle  4\right\rangle=\\{[4][8]\\}$ con ordine 2.
- $\left\langle  8\right\rangle=\\{8\\}$ con ordine 1.

Ovviamente esistono poi tutti gli altri sottogruppi con ordine pari ad $8$  che sono generati da elementi  di $\mathbb{Z}_8$ coprimi. Possiamo quindi definire la seguente applicazione che dimostreremo essere un omomorfismo:

$$\varphi_d :\begin{array}{rcl}
\mathbb{Z}_8 & \longrightarrow & \mathbb{Z}_8 \\
\varphi_d ([z]) & \longrightarrow &  [dz]
\end{array}\\qquad \text{ con $d\in\mathbb{Z}_8$}$$

L'applicazione appena definita ha molto in comune con gli isomorfismi $\phi_u\in Aut(\mathbb{Z}_n)$ studiati in #150 , in quello specifico caso  avevamo che $u\in \mathcal{U}(\mathbb{Z}_n)$ e quindi ogni $u$ era generatore dell'intero insieme. In questo caso ci accontentiamo anche di applicazioni che non mappano tutto $\mathbb{Z}_8$ , ovvero tutti gli endomorfismi del gruppo. Dimostriamo ora che $\varphi_d$  è un omomorfismo.

$$\varphi_d(z_1+z_2)=d(z_1+z_2)=dz_1+dz_2=\varphi_d(z_1)+\varphi_d(z_2)$$

Essendo un omomorfismo $\varphi_d(n)=[n]$.
Quindi $Aut(\mathbb{Z}_n)\subset End(\mathbb{Z}_n)$, in particolare l'omomorfismo da noi definito  copre l'intero insieme. 

-------------------

Dobbiamo ora occuparci del $Ker\varphi_d$, che ci servirà per definire il gruppo quoziente.  Per i nostri scopi è utile scoprire che cardinalità ha questo sottogruppo. All'atto pratico quando noi scriviamo la mappa di $\varphi_d$ per $\mathbb{Z}_8$ passiamo su ogni elemento del sottogruppo  $\frac{o(\mathbb{Z}_n)}{o(d)}$ volte prima di esaurire gli elementi del gruppo di partenza ( il modo in cui ho trovato la cardinalità  è da rivedere, in quanto abbastanza pratico e poco collegato con la struttura dei due gruppi) , ne segue che $|Ker\varphi_d|=\frac{n}{o(d)}$ , dove $o(d)$ è l'ordine del sottogruppo generato da $d$. 

------------------
Ora che abbiamo definito il nucleo ed il numero dei suoi elementi possiamo utilizzare il Teorema fondamentale dell'omomorfismo per  definire i gruppi quoziente. Sappiamo quindi che esistono gruppi quoziente di $\mathbb{Z}_8$ (stesso discorso per un $n$ generico), definiti da tutti le $\varphi_d\in End(\mathbb{Z}_8)$. Prima di procedere con i vari gruppi quoziente, sempre grazie al Teo. Fond. Omo.  conosciamo la cardinalità del gruppo quoziente, dato che:
$$\mathbb{Z}_n/Ker\varphi_d\simeq Im(\varphi_d)$$
$$|\mathbb{Z}_n/Ker\varphi_d|=|Im(\varphi_d)|=o(d)=\frac{n}{n}\cdot o(d)=\frac{|\mathbb{Z}_n|}{|Ker\varphi_d|}$$

----------------------------------
Definiamo ora tutti i gruppi  quoziente di $\mathbb{Z}_8$:

$$\mathbb{Z}_8/Ker\varphi_1 \simeq \mathbb{Z}_8/Ker\varphi_3 \simeq \mathbb{Z}_8/Ker\varphi_5 \simeq \mathbb{Z}_8/Ker\varphi_7 \simeq \mathbb{Z}_8 \simeq Im(\varphi_u)$$

In generale i gruppi quoziente dati dal nucleo di automorfismi sono tutti isomorfi tra loro ed isomorfi a $\mathbb{Z}_8$ stesso.

$$\mathbb{Z}_8/Ker\varphi_2\simeq \mathbb{Z}_8/Ker\varphi_6 \simeq Im(\varphi_2) \simeq Im(\varphi_6)\simeq  \left\langle  2\right\rangle$$

$$\mathbb{Z}_8/Ker\varphi_4\simeq Im(\varphi_4) \simeq \left\langle  4\right\rangle$$

$$\mathbb{Z}_8/Ker\varphi_8\simeq Im(\varphi_8) \simeq \left\langle  8\right\rangle$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/03a3dda9-7b9c-4955-8a01-8e67b3de9d47)
